### DISCUSSION TITLE
Potential issue on httpx.url and partial url parsing

### DISCUSSION THREAD

**Discussion by nkpf42** [encode/httpx#3369] [Potential Issue] [answered]
Using httpx.url to parse a partial URL returns incorrect parsing

In my case the character `+` is transformed to a space. We can see the wrong escaped character `%20` instead of `%2B`

```
In [41]: my_url = httpx.URL("my_url?date_time=2024-10-28T09:09:27+00:00")

In [42]: my_url.params
Out[42]: QueryParams('date_time=2024-10-28T09%3A09%3A27%2000%3A00')

In [43]: urllib.parse.unquote(str(my_url.params))
Out[43]: 'date_time=2024-10-28T09:09:27 00:00'
```

**Comment by lovelydinosaur** [comment, marked as answer]
Thanks yep. Resolved in #3373.

**Reply by nkpf42** [reply]
Thanks a lot for your quick fix :slightly_smiling_face: When can I expect a patch release for this one ? (I don't know your release cycle policy)
